linera-chain/src/lib.rs

```diff
@@ -106,12 +106,10 @@ pub enum ChainError {
     },
     #[error("The signature was not created by a valid entity")]
     InvalidSigner,
-    #[error(
-        "Was expecting block height {expected_block_height} but found {found_block_height} instead"
-    )]
+    #[error("Chain is at height {chain_tip_height} but block is at {block_height}")]
```

Comment:
How about keeping the current fields and adding one for the chain:
`Chain {chain_id} is expecting a next block at height {expected_block_height} but the given block is at height {found_block_height} instead`

Reply:
Mmm this won't be backportable though. Also I wonder if we should have a separate error from block proposals and one for confirmed blocks.

Reply:
Actually: I'll revert the field renaming, and _not_ add the chain ID yet. That way we can cleanly backport it to Conway.